tekton/release-cheatsheet.md

```diff
@@ -0,0 +1,142 @@
+   and note the commit's hash.
+
+1. Create environment variables for bash scripts in later steps.
+
+    ```bash
+    VERSION_TAG=# UPDATE THIS. Example: v0.5.0
+    RELEASE_GIT_SHA=# SHA of the release to be released
+    ```
+
+1. Confirm commit SHA matches what you want to release.
+
+    ```bash
+    git show $RELEASE_GIT_SHA
+    ```
+
+1. Create a workspace template file:
+
+   ```bash
+   cat <<EOF > workspace-template.yaml
+   spec:
+     accessModes:
+     - ReadWriteOnce
+     resources:
+       requests:
+         storage: 1Gi
+   EOF
+   ```
+
+1. Execute the release pipeline.
+
+
+    ```bash
+        tkn --context dogfooding pipeline start results-release \
+        --serviceaccount=results-release \
+        --param=revision="${RELEASE_GIT_SHA}"  \
+        --param=version="${VERSION_TAG}"  \
+        --param=bucket=gs://tekton-releases/results \
+        --workspace name=release-secret,secret=release-secret \
+        --workspace  name=ws,volumeClaimTemplateFile=workspace-template.yaml
+    ```
+
+1. Watch logs of result-release.
+
+1. Once the pipeline run is complete, check its results:
+
+   ```bash
+   tkn --context dogfooding pr describe <pipeline-run-name>
+
+   (...)
+   📝 Params
+
+   NAME                    VALUE
+   revision                 6ea31d92a97420d4b7af94745c45b02447ceaa19
+
+   (...)
+   ```
+
+   The `commit-sha` should match `$RELEASE_GIT_SHA`.
+   The two URLs can be opened in the browser or via `curl` to download the release manifests.
+
+    1. The YAMLs are now released! Anyone installing Tekton Results will now get the new version. Time to create a new GitHub release announcement:
+
+    1. Create additional environment variables
+
+    ```bash
+    OLD_VERSION=# Example v0.4.0
+    TEKTON_PACKAGE=tektoncd/results
+    ```
+
+
+    1. Execute the Draft Release task.
+
+    ```bash
+    tkn --context dogfooding pipeline start   \
+    --workspace name=shared,volumeClaimTemplateFile=workspace-template.yaml    \
+    --workspace name=credentials,secret=release-secret  \
+    -p package="${TEKTON_PACKAGE}"     -p git-revision="${RELEASE_GIT_SHA}"  \
+    -p release-tag="${VERSION_TAG}"     -p previous-release-tag="${OLD_VERSION}"   \
+    -p release-name="Tekton Results"     -p bucket="gs://tekton-releases/results"   \
+    -p rekor-uuid="$REKOR_UUID"  --dry-run    release-draft
+    ```
+
+    1. Watch logs of create-draft-release
+
+    1. On successful completion, a URL will be logged. Visit that URL and look through the release notes.
+      1. Manually add upgrade and deprecation notices based on the generated release notes
+      1. Double-check that the list of commits here matches your expectations
+         for the release. You might need to remove incorrect commits or copy/paste commits
+         from the release branch. Refer to previous releases to confirm the expected format.
+    1. Un-check the "This is a pre-release" checkbox since you're making a legit for-reals release!
```

Comment:
The actual message of the checkbox is "Set as a pre-release". It's easy to figure, but easy to fix the doc.

Reply:
Change the message to the correct one.